scripts/increment_simple.lua

```diff
@@ -0,0 +1,20 @@
+local arg_incrby = tonumber(ARGV[1])
+local arg_max = tonumber(ARGV[2])
+
+local old_value = redis.call("PTTL", KEYS[1])
+if old_value == -1 then -- key exists but has no associated expire
+   return -1 -- -1 signals key exists but has no associated expire
+elseif old_value == -2 then -- key does not exist
+   if arg_incrby > arg_max then
```

Comment:
This should be in ruby code.

Reply:
Good point. I was trying to be defensive, but it doesn't make sense here.